compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/NodeParser.java

```diff
@@ -125,4 +125,15 @@ public static TypeDescriptorNode parseTypeDescriptor(String text) {
         BallerinaParser parser = ParserFactory.getParser(text);
         return parser.parseAsTypeDescriptor().createUnlinkedFacade();
     }
+
+    /**
+     * Parses the input as an object member.
+     *
+     * @param text the input
+     * @return a {@code TypeDescriptorNode}
```

Comment:
return type is wrong.
```suggestion
     * @return a {@code Node}
```